### DISCUSSION TITLE
Eslint 👉 `import/no-unresolved` for `nuxt@3.7.3`

### DISCUSSION THREAD

**Discussion by albertcito** [nuxt/nuxt#23344] [Questions] [answered]
I used `nuxt@3.6.2` and updated it to `nuxt @3.7.3` and I'm getting the `eslint` error: `Unable to resolve path to module @/xyz/abc   import/no-unresolved`.

`tsconfig` file for `nuxt@3.6.2` has 

```json
"baseUrl": "/Users/user/Documents/project",
"@": [	
   "/Users/user/Documents/project"	 
],
```

`tsconfig` file for `nuxt @3.7.3` doesn't have `baseUrl` and the path for `@` is 

```json
"@": [	
    "../*"
 ],
```

So for that reason I'm getting those `eslint` issues. Any of you know how to fix is please?

**Comment by mattmess1221** [comment, marked as answer]
The solution is to configure your eslintrc with the location of the nuxt generated `tsconfig.json` file.

Via https://github.com/nuxt/nuxt/issues/22994#issuecomment-1709053148
```json
// .eslintrc
{
  "settings": {
    "import/resolver": {
      "typescript": {
        "project": "./.nuxt/tsconfig.json"
      }
    }
  }
}
```

**Reply by igor-boiko-liven** [reply]
Hi
How can i do it for flat eslint v9 config? Because it doesnt have settings key inside...
